packages/components/src/InputFile/InputFile.tsx

```diff
@@ -101,6 +119,9 @@ interface InputFileProps {
 }
 
 export function InputFile({
+  variation = "dropzone",
+  size = "base",
+  buttonLabel: providedButtonLabel,
```

Comment:
A lack of knowledge here, how does this work? what is `providedButtonLabel`?

Reply:
That renames the value that it is destructured into.